geowebcache/pom.xml

```diff
@@ -53,32 +53,43 @@
   
   <repositories>
     <repository>
-      <id>boundless</id>
-      <name>Boundless Maven Repository</name>
-      <url>https://repo.boundlessgeo.com/main/</url>
+      <id>osgeo-release</id>
+      <name>OSGeo Repository</name>
+      <url>https://repo.osgeo.org/repository/release/</url>
       <snapshots>
-        <enabled>true</enabled>
+        <enabled>false</enabled>
       </snapshots>
+      <releases>
+        <enabled>true</enabled>
+      </releases>
     </repository>
     <repository>
-      <id>osgeo</id>
-      <name>Open Source Geospatial Foundation Repository</name>
-      <url>https://download.osgeo.org/webdav/geotools/</url>
+      <id>osgeo-snapshot</id>
+      <name>OSGeo Repository</name>
+      <url>https://repo.osgeo.org/repository/snapshot/</url>
+      <!-- Contains releases for OSGeo projects: GeoTools, GeoWebCache, GeoServer, ...     -->
+      <!-- Includs third-party-dependences, and caches of:                                 -->
+      <!-- ucar (https://artifacts.unidata.ucar.edu/content/repositories/unidata-releases) -->
+      <!-- geosolutions (http://maven.geo-solutions.it/)                                   -->
```

Comment:
Aren't these only in the release repository, not the snapshot one?

Also, I don't believe GeoWebCache cares about either of those (maybe geosolutions?), so there isn't much of a reason to mention them